docs/admin/users/oidc-auth.md

```diff
@@ -11,16 +11,7 @@ Your OIDC provider will ask you for the following parameter:
 
 ## Step 2: Configure Coder with the OpenID Connect credentials
 
-Navigate to your Coder host and run the following command to start up the Coder
-server:
-
-```shell
-coder server --oidc-issuer-url="https://issuer.corp.com" --oidc-email-domain="your-domain-1,your-domain-2" --oidc-client-id="533...des" --oidc-client-secret="G0CSP...7qSM"
-```
-
-If you are running Coder as a system service, you can achieve the same result as
-the command above by adding the following environment variables to the
-`/etc/coder.d/coder.env` file:
+Set the following environment variables on your Coder deployment and restart Coder:
```

Comment:
optional

```suggestion
Set the following environment variables in your Coder deployment's `/etc/coder.d/coder.env` file, then restart Coder:
```

Reply:
What I am trying to do is make all the instructions common. The `env` file doesn't apply for K8 s and docker based installations.